traits/has_traits.py

```diff
@@ -2649,10 +2649,31 @@ def sync_trait ( self, trait_name, object, alias = None, mutual = True,
 
             return
 
-        value = ( weakref.ref( object, self._sync_trait_listener_deleted ),
-                  alias )
-        dic   = self._get_sync_trait_info().setdefault( trait_name, {} )
+        # Callback to use when the synced object goes out of scope. In order
+        # to avoid reference cycles, this must not be a member function. See
+        # Github issue #69 for more detail.
+
+        # FIXME: This callback removes the synced object from the
```

Comment:
Is there an issue open for this?  If so, please reference the issue from the comment.  If not, please make one. :-)